### PR TITLE
Notes on deploying Zeppelin on Kubernetes

### PR DESCRIPTION
**Pull request by stvoutsin**
Added some notes addressing issue #107 (Deploy Zeppelin on Kubernetes)

### CODE REVIEW

File: notes/20200525-zeppelin-k8s.txt [hunk 1]
```diff
@@ -0,0 +1,143 @@
+
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2019, ROE (http://www.roe.ac.uk/)
```
Comment (Zarquan) [inline notes/20200525-zeppelin-k8s.txt:4]: 2020

File: notes/stv/20200525-zeppelin-k8s.txt [hunk 2]
```diff
@@ -0,0 +1,143 @@
+
+# <meta:header>
+#   <meta:licence>
+#     Copyright (c) 2019, ROE (http://www.roe.ac.uk/)
```
Comment (Zarquan) [inline notes/stv/20200525-zeppelin-k8s.txt:4]: 2020

File: notes/stv/20200526-zeppelin-k8s.txt [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+# ----------------------------------------
+# Deploying Zeppelin on K8s
+# ----------------------------------------
+
+The following assumes a running K8s cluster on Openstack, creates using following instructions:
+https://raw.githubusercontent.com/stvoutsin/aglais/stv-issue-19/notes/stv/20200408-Spark-on-K8s.txt
+
+For this first experiment, we are deploying on a cluster that has Hadoop & Spark installed.
+
+
```
Comment (Zarquan) [inline notes/stv/20200526-zeppelin-k8s.txt:34]: Missing a step that installs Helm ?

**Comment by stvoutsin** [comment]
I've removed the file that was added under notes by mistake and also added the instructions that were run to install Helm

**Pull request merged**